Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM rust:1.43.1 as build
+ENV PKG_CONFIG_ALLOW_CROSS=1
+
+WORKDIR /usr/src/labrinth
+# Download and compile deps
+COPY Cargo.toml .
+COPY Cargo.lock .
+COPY docker_utils/dummy.rs .
```

Comment:
maybe just `echo "fn main() {}" > dummy.rs`? One less useless file in git

Reply:
It's a possibility, but that means that we have to edit the docker file to fix a dependency compilation inconsistency, so break the whole caching system